gef-ui/server/controllers/confirm-cover-start-date/index.js

```diff
@@ -30,6 +35,8 @@ const updateCoverStartDate = async (facilityId, { coverStartDate, shouldCoverSta
 
 const processCoverStartDate = async (req, res) => {
   const { dealId, facilityId } = req.params;
+  const { user } = req.session;
+  const { _id } = user;
```

Comment:
Similar to above, if we call _id `userId` it'll make it clearer in the functions below. Feels a bit strange having `dealId, _id` down below

Reply:
done